### ISSUE TITLE
You get a 404 error code when you try to synchronize ?

### ISSUE THREAD

**Issue by murelh** [hiboutik/woocommerce-hiboutik#3]
Hi!

If you get a a 404 error code when you try to synchronize, it ay be because your Wordpress is using  an incompatible URL structure.
Hiboutik plugin is proven to work fine with permalinks based on publication titles.

![2019-05-31 09_43_44-Réglages des permaliens ‹ — WordPress - Iron](https://user-images.githubusercontent.com/2480783/58690077-02aa8a80-8389-11e9-9e13-f6283798813c.png)

For now, custom url can definitely leads to 404 pages.

Hope this help.
Good dev

**Comment by floristeriaellenguajedelasflores** [comment]
Hi! I am trying to fix this problem but in my Woocommerce I have different information. I have URL address and Site Address. I have not titre, structure... etc. How can I fix this? I got the 404 error code. Can you  please help me to fix?
![Untitled](https://user-images.githubusercontent.com/83462474/116713940-889c3980-a9d5-11eb-9a25-aac31e581150.jpg)

**Comment by murelh** [comment]
Hello, does this help you ?
https://github.com/hiboutik/woocommerce-hiboutik/issues/3

All WP websites should have a section that allows you to configure your link structure.
If updating the URL by the recommended option works, then this is just it.

If not, tell me more

Regards,
Murelh Ntyandi

 Cordialement,

***@***.***D73DE8.AE6BDB10]
Murelh Ntyandi
Développeur Web PHP Full-stack
https://murelh.info<https://murelh.info/>
+33 (0)6.67.85.94.47



De : floristeriaellenguajedelasflores ***@***.***>
Envoyé : vendredi 30 avril 2021 17:01
À : hiboutik/woocommerce-hiboutik ***@***.***>
Cc : murelh ***@***.***>; Author ***@***.***>
Objet : Re: [hiboutik/woocommerce-hiboutik] You get a 404 error code when you try to synchronize ? (#3)


Hi! I am trying to fix this problem but in my Woocommerce I have different information. I have URL address and Site Address. I have not titre, structure... etc. How can I fix this? I got the 404 error code. Can you please help me to fix?
[Untitled]<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F83462474%2F116713940-889c3980-a9d5-11eb-9a25-aac31e581150.jpg&data=04%7C01%7C%7Ca60fee0b251a400223d808d90be8bb78%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637553916469311644%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=oJL33zycC87M4Mjt6SCje%2Br4pLf1rNjElWlzWeRgSus%3D&reserved=0>

-

**Comment by floristeriaellenguajedelasflores** [comment]
No, this is not working. I did the WP AP section fine. But when I put the code I have the error 404. 
According to your screenshot, I must have something called "titre de la publication" but this is not existing in WP anymore, I have the information I sent to you in my screenshot. 
How can I solve this? AP is included in Hiboutik but I have 404 error code.

**Comment by murelh** [comment]
Are you looking at Ajustes -> Enlaces permanentes ?
I am running an other WP (V5.7.1) and there is still this url link with post name in permalink settings.
In spanish it is called : « Nombre de la entrada »

Hoping i twill make it
Regards,


De : floristeriaellenguajedelasflores ***@***.***>
Envoyé : vendredi 30 avril 2021 17:53
À : hiboutik/woocommerce-hiboutik ***@***.***>
Cc : murelh ***@***.***>; Author ***@***.***>
Objet : Re: [hiboutik/woocommerce-hiboutik] You get a 404 error code when you try to synchronize ? (#3)


No, this is not working. I did the WP AP section fine. But when I put the code I have the error 404.
According to your screenshot, I must have something called "titre de la publication" but this is not existing in WP anymore, I have the information I sent to you in my screenshot.
How can I solve this? AP is included in Hiboutik but I have 404 error code.

-

**Comment by floristeriaellenguajedelasflores** [comment]
oh! You are right! I copied the link from "nombre de la entrada" to URL in the WooCommerce of Hiboutik + ck_ + cs_and I still have the same issue: 
L'API Woocommerce n'est pas correctement configurée
JSON ERROR: Syntax error (code 404)
Check your server logs
https://github.com/hiboutik/woocommerce-hiboutik/issues/3

**Comment by floristeriaellenguajedelasflores** [comment]
Can you please help me with this error? 
I have selected the "titre de la publication" as URL in WP (already selected before)
Which URL do I need to complete in URL space in tocket for Hiboutik?
I copied also the API codes. 
Best regards

**Comment by murelh** [comment]
> Can you please help me with this error?
> I have selected the "titre de la publication" as URL in WP (already selected before)
> Which URL do I need to complete in URL space in tocket for Hiboutik?
> I copied also the API codes.
> Best regards

Hi,

Sorry for the late answer.
It is write hard to tell without having a look on your WP.
If you want to, contact me to set up an appointment .
I do need a view on the installed plugin and Hiboutik.

Before all that, I have to ask if you did read the official documentation at : http://www.pos-software-free.com/sync-woocommerce-hiboutik/

Regads,
Murelh
